web version/app.py

```diff
@@ -36,20 +36,25 @@ def allowed_image_size(filesize):
     return True
 
 
+@app.route('/<string:page_name>')
+def html_page(page_name):
+    return render_template(page_name)
+
+
 @app.route('/', methods=['GET', 'POST'])
 def img_upload():
     if request.method == "POST":
         if request.files:
             if "filesize" in request.cookies:
                 if not allowed_image_size(request.cookies["filesize"]):
-                    print("filesize exceeded max limit")
-                    return redirect(request.url)
+                    flash("filesize exceeded max limit")
+                    return redirect('error-page.html')
```

Comment:
I think we can accept redirecting away for now. I say this because someone might complain about it. But we can update later it to show messages on the same page.

Reply:
updated to show messages on the same page